### PR TITLE
Improve requests

### PR DESCRIPTION
**Pull request by dkuang1980**

### CODE REVIEW

File: wukong/request.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
 from wukong.zookeeper import Zookeeper
-from requests.exceptions import ConnectionError
 from wukong.errors import SolrError
 import requests
 import random
 import json
 import time
+import logging
+
+log = logging.getLogger(__name__)
```
Comment (TylerLubeck) [inline wukong/request.py]: I think in other files we use `logger = logging.getLogger(__name__)`. Perhaps worth changing it for the sake of consistency?

Comment (dkuang1980) [reply wukong/request.py]: yes, we should change it

File: setup.py [hunk 2]
```diff
@@ -13,7 +13,7 @@ def read(fname):
     README = read('README.md')
     CHANGES = read('CHANGES.md')
 
-version = '0.2.2'
+version = '0.3.0'
```
Comment (TylerLubeck) [inline setup.py]: Check your versioning? I think this is 0.4.0

**Comment by TylerLubeck** [comment]
Closing in favor of #18

**Pull request closed without merging**